src/psyclone/tests/psyir/tools/call_tree_utils_test.py

```diff
@@ -441,3 +448,90 @@ def test_call_tree_utils_const_argument():
     # Make sure the constant '0' is not listed
     assert "0" not in read_write_info.signatures_read
     assert Signature("0") not in read_write_info.signatures_read
+
+
+# -----------------------------------------------------------------------------
+@pytest.mark.usefixtures("clear_module_manager_instance")
+def testcall_tree_utils_non_local_inout_parameters(capsys):
+    '''Tests the collection of non-local input and output parameters.
+    '''
+    Config.get().api = "dynamo0.3"
```

Comment:
This should perhaps use monkeypatch (unless we have a fixture that automatically resets Config after every test)?

Reply:
We do have one.